Tests/TABResourceLoaderTests/Helpers/CharacterSet+Additions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  CharacterSet+Additions.swift
+//  TABResourceLoader
+//
+//  Created by Marco Guerrieri on 14/03/2019.
+//  Copyright © 2019 Luciano Marisi. All rights reserved.
+//
+
+import Foundation
+
+extension CharacterSet {
+  public static var improvedUrlQueryAllowed: CharacterSet {
+    var characterSet = CharacterSet.urlQueryAllowed
+    characterSet("&")
```

Comment:
You're missing a call to `remove()`

Reply:
That's nice, I also built but not noticed. Thank you